### DISCUSSION TITLE
yarn start issue

### DISCUSSION THREAD

**Discussion by ghost** [FlowiseAI/Flowise#1147] [Q&A]
PS D:\Dhanu\LLAMA\Flowise> yarn start
yarn run v1.22.19
$ run-script-os
$ cd packages/server/bin && run start
 »   Error: command start not found
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command

**Comment by arberrexhepi** [comment]
i've had the same issue, so i've just been using npm run start instead, use yarn to install and build then use npm run start, works

**Reply by ghost** [reply]
Thanks for reply, you said try with npm now i tried with npm i got this error 

PS D:\Dhanu\LLAMA\Flowise> npm flowise start
Unknown command: "flowise"

To see a list of supported npm commands, run:
  npm help
PS D:\Dhanu\LLAMA\Flowise> npx flowise start
 »   Error: command start not found

**Reply by arberrexhepi** [reply]
well you didn't use the same command as I mentioned, i did "npm run start", you tried "npm flowise start".. not sure what to tell you

**Reply by ghost** [reply]
This is the flow i followed 

## ⚡Quick Start

Download and Install [NodeJS](https://nodejs.org/en/download) >= 18.15.0
1. Install Flowise
    ```bash
    npm install -g flowise
    ```
2. Start Flowise

    ```bash
    npx flowise start
    ```
    With username & password
    ```bash
    npx flowise start --FLOWISE_USERNAME=user --FLOWISE_PASSWORD=1234
    ```
3. Open [http://localhost:3000](http://localhost:3000)
